Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs

```diff
@@ -3,8 +3,10 @@
 
 using Microsoft.MixedReality.Toolkit.Utilities.Editor;
 using System.IO;
+using System.Linq;
 using UnityEditor;
 using UnityEngine;
+using UnityEngine.Experimental.GlobalIllumination;
```

Comment:
Are these new `using`s used?

Reply:
i do not believe so... i will remove them